src/ext/compatibility.h

```diff
@@ -3,6 +3,7 @@
 
 #include <TSRM/TSRM.h>
 #include <Zend/zend.h>
+#include <php_version.h>
```

Comment:
Why do we need this include?

Reply:
This is what adds the `PHP_VERSION_ID` definition. My guess is that most files which include this one also include `<php.h>` or some other file previously, so in the past we didn't have issues. However, from this particular file there isn't such an include, so we need the proper header.